app/views/hyrax/my/_facet_limit.html.erb

```diff
@@ -1,4 +1,4 @@
-  <ul class="dropdown-menu facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
+  <ul class="<%= 'dropdown-menu ' if request.format.html? %>facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
```

Comment:
This template is also used by AJAX requests to render facets in a modal panel. In that case, the request format is `json`.